src/NServiceBus.Transport.SQS.Tests/InputQueuePumpTests.cs

```diff
@@ -115,7 +115,7 @@ public async Task Expired_messages_are_deleted_without_processing()
                     {Headers.MessageId, messageId},
                     {TransportHeaders.TimeToBeReceived, ttbr.ToString()}
                 },
-                Body = "empty message"
```

Comment:
Do we still need to support this type of "empty" message? If so, should we duplicate this test rather than modifying it? Or is the body irrelevant to this test?

Reply:
The body is irrelevant but I have used string.Empty and documented the reasoning here: https://github.com/Particular/Platform/pull/446